.buildkite/hooks/pre-exit

```diff
@@ -2,6 +2,16 @@
 
 set -eo pipefail
 
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent" && "$BUILDKITE_STEP_KEY" == "integration-tests" ]]; then
+  if [[ -z "${WORKSPACE-""}" ]]; then
+      WORKSPACE=$(git rev-parse --show-toplevel)
+  fi
+  source "${WORKSPACE}/.buildkite/scripts/common.sh"
+
+  # Perform cleanup of integration tests resources
+  TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 SNAPSHOT=true mage integration:clean
```

Comment:
Can we add an extra log line with the `--- ` prefix so that in Buildkite it logs nicely and we know when this is executed?
Maybe something along the lines of 
```suggestion
  echo "--- Cleaning up integration test resources"
  TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 SNAPSHOT=true mage integration:clean
```

Reply:
Good idea!
Most of the time the output will be empty if the cleanup already completed in the step, so having the log printed is convenient :D

Reply:
Could we place the `TEST_INTEG_AUTH_ESS_REGION=azure-eastus2` in the `common.sh` for both the testing step and this pre-exit? That way it only needs to be adjusted in one place.